spec/Candid.md

```diff
@@ -711,7 +905,33 @@ not (null <: <datatype>)
 <datatype> <: <datatype'> ~> f
 ------------------------------------------
 <datatype> <: opt <datatype'> ~> \x.?(f x)
+
+not (<datatype> <: <datatype'>)
+---------------------------------
+opt <datatype> <: opt <datatype'>
+  ~> \x.join_opt (\y.
```

Comment:
`join_opt` is a bit obscure. maybe write it out with `match … with`, for greater clarity?

Reply:
Done.